app/src/ui/add-repository/create-repository.tsx

```diff
@@ -265,10 +280,62 @@ export class CreateRepository extends React.Component<ICreateRepositoryProps, IC
     )
   }
 
+  private renderInvalidPathWarning() {
+    const isValidPath = this.state.isValidPath
+
+    if (isValidPath === null || isValidPath) {
```

Comment:
Could we extract the `isValidPath === null` check to a more friendly name, so we can understand that we're only really interested in `false` here for rendering the error?